internal/stack/localresources.go

```diff
@@ -104,3 +117,24 @@ func esHostWithPort(host string) string {
 
 	return host
 }
+
+func DockerInternalHost(host string) string {
+	url, err := url.Parse(host)
+	if err != nil {
+		return host
+	}
+
+	ip := net.ParseIP(url.Hostname())
+	if url.Hostname() == "localhost" || (ip != nil && ip.IsLoopback()) {
+		const hostInternal = "host.docker.internal"
```

Comment:
This is a special DNS name used by Docker desktop, would it cause any issue with users running elastic-package on Mac ?
https://docs.docker.com/desktop/features/networking/#i-want-to-connect-from-a-container-to-a-service-on-the-host

Reply:
I am explicitly adding the host in the docker compose with the hope it works the same on all platforms: https://github.com/elastic/elastic-package/pull/2298/files#diff-379a1fb4d0ceacb95a5ce10db57b58cf8c43aa694f7576a1f175f01ef507392fR65

Though I haven't tested it in Windows or Mac yet.

If there is some conflict with docker desktop maybe another option is to use just another host.